frontend/apps/reader/modules/readergoto.lua

```diff
@@ -43,8 +43,9 @@ function ReaderGoto:onShowGotoDialog()
     else
         input_hint = T("@%1 (1 - %2)", curr_page, self.document:getPageCount())
     end
+    input_hint = input_hint .. string.format("  %.2f%%", curr_page / self.document:getPageCount() * 100)
     self.goto_dialog = InputDialog:new{
-        title = _("Enter page number"),
+        title = _("Enter page number or %"),
```

Comment:
Is there a specific rationale behind not writing percentage?

Reply:
Can we keep `Go to %` in the button (lack of space)?

Reply:
Of course. But if the rationale is just that it's shorter, no need in the title. ^_^